### ISSUE TITLE
Save layout yields blank html

### ISSUE THREAD

**Issue by itsgifnotjiff** [holoviz/panel#6806]
As discussed in [this question](https://discourse.holoviz.org/t/allow-user-to-download-parts-of-the-app-as-static-html/7188/2?u=stuckduckf) with Mr. @MarcSkovMadsen , there seems to be a bug with `.save`.

Here is a MRE that tries to save a dynamically generated Holoviews populated Panel layout. I am trying to get this to work so I can save dynamically generated Holoviews plots based on user selection on a huge dataset. Idea being I can export only the statistics plot and not the original data dynamically (this would kill many operational DBs around the world I assume).

```python
import panel as pn
import holoviews as hv
import numpy as np
from bokeh.resources import INLINE
import io

ACCENT_COLOR = "#308752"

pn.extension(
    sizing_mode="stretch_width",
    notifications=True
)

class ButtonOnClick(pn.viewable.Viewer):
    
    def _load_callback(self, event):
        points = hv.Points(np.random.randn(1000, 2))
        selection = hv.streams.Selection1D(source=points)
        def selected_info(index):
            arr = points.array()[index]
            if index:
                label = 'Mean x, y: %.3f, %.3f' % tuple(arr.mean(axis=0))
            else:
                label = 'No selection'
            return points.clone(arr, label=label).opts(color='red')

        selected_points = hv.DynamicMap(selected_info, streams=[selection])
        layout = points.opts(tools=['box_select', 'lasso_select']) + selected_points
        self.figs_col.append(pn.pane.HoloViews(layout))
        pn.state.notifications.success('_load_callback')

    def _download_callback(self):
        # pn.save(self.figs_col, 'test.html', resources=INLINE)
        # self.figs_col.save('test.html')
        pn.state.notifications.success('_download_callback')
        sio = io.StringIO()
        self.figs_col[0].save(sio)
        sio.seek(0)
        return sio
    
    def __init__(self):
        self.figs_col = pn.Column()
        self.load_button = pn.widgets.Button(name="Load")
        self.load_button.on_click(self._load_callback)
        self.download_button = pn.widgets.FileDownload(filename='layout.html', callback=self._download_callback)
        super().__init__()
        
        
    def view(self):
        return self.download_button

app = ButtonOnClick()

template = pn.template.FastListTemplate(
    accent_base_color=ACCENT_COLOR,
    header_background=ACCENT_COLOR,
    title="Button Click Callback Test",
    main=[app.load_button, app.download_button, app.figs_col],
    sidebar_width=200,
    theme="dark",
).servable()
```

**Comment by Coderambling** [comment]
Is the below related?

https://discord.com/channels/1075331058024861767/1088157184489164831/1233798917485760656

Also see here: https://discourse.holoviz.org/t/button-on-click-callback-never-called/7180

**Comment by philippjfr** [comment]
So this is caused by the fact that the component is styled with the Fast design. I've pushed a fix in #6832 for now but we likely need to improve on that a bit. That said when exporting components that are already rendered you're going to be inheriting a lot of the styling from the app itself which may not be appropriate. I'd probably suggest recreating the plot for specifically for export.
